### ISSUE TITLE
Notification message and title NOT replaced in persistent notifications

### ISSUE THREAD

**Issue by stezak77** [home-assistant/android#3149]
**Home Assistant Android version:**
2022.11.0-full

**Android version:**
13

**Phone model:**
Samsung Galaxy A53 5G

**Home Assistant version:**
2022.11.5

**Description**
I have an Android Samsung Galaxy A53 5G smartphone with latest Home Assistant companion app installed.

When Home Assistant sends a persistent notification to my phone, properly using `tag` and `persistent` service call attributes, notification **message and title are NOT replaced**. Surprisingly, action titles are.

### First notification:

Service call:
```
service: notify.my_smartphone
data:
  message: Message One
  data:
    tag: persistent
    persistent: true
    actions:
      - action: dummy
        title: Action One
  title: Title One
```
Notification displayed on smartphone:
![image](https://user-images.githubusercontent.com/49878007/207343824-4712b95e-5a25-42d9-a1cc-49f9860c4943.png)

Notification log in companion app:
![image](https://user-images.githubusercontent.com/49878007/207344038-b8df4859-80c5-4901-8aab-c22cd84bd7b8.png)

### Second notification:

Service call:
```
service: notify.my_smartphone
data:
  message: Message Two
  data:
    tag: persistent
    persistent: true
    actions:
      - action: dummy
        title: Action Two
  title: Title Two
```
Notification displayed on smartphone - message and title are NOT replaced, action title is:
![image](https://user-images.githubusercontent.com/49878007/207344259-45dbd229-d752-426c-9a07-f79fd6b37da0.png)

Notification log in companion app - here everything looks correct:
![image](https://user-images.githubusercontent.com/49878007/207344325-c197bbd8-0406-4001-a9b5-e1757611838f.png)

**Comment by dshokouhi** [comment]
closing as duplicate of #3128

**Comment by stezak77** [comment]
Issue solved after updated to One UI 5.1 on my Samsung Galaxy A53 5G
